data/pokedex.ts

```diff
@@ -162,6 +162,7 @@ export const Pokedex: {[speciesid: string]: SpeciesData} = {
 		weightkg: 0,
 		color: "Red",
 		eggGroups: ["Monster", "Dragon"],
+		changesFrom: "Charizard",
```

Comment:
These are supposed to be filled in automatically from `getSpecies`, right? Or did we change that?

Reply:
We changed it